### PR TITLE
Updated title pages

### PR DESCRIPTION
**Pull request by sbecuwe**
- img/logo_vsc_2.pdf (new)
  - new VSC logo

- intro-HPC/intro-HPC.tex
  - moved "Fine-tuning Job Specifications" to the Advanced part: it was already
    shown in that table, but physically still part of the Basics part
  - rewritten the \if construction around \include{ch_checkpointing} for
    gent and brussel
  - removed \include{ch_VNC} from the antwerpen documentation

- intro-HPC/title.tex
  - more uniform title page, including new VSC logo
  - made the tables for the three parts consistent with the table of contents
    (same order)
  - introduced \ifthenelse construction to exclude specific chapters for
    specific sites (be careful with \\ \hline !)

- intro-Linux/title.tex
  - more uniform title page, including new VSC logo

- perfexpert/title.tex
  - more uniform title page, including new VSC logo

- Makefile
  - added mrproper option to remove generated pdf files too

### CODE REVIEW

**Comment by sbecuwe** [comment]
Suggestion related to this commit: we could consider moving authors and affiliations to the second page to have a "clean" title page.

File: intro-HPC/intro-HPC.tex [hunk 1]
```diff
@@ -78,20 +78,20 @@ \part{Advanced Guide}
...
+}{}
 
 \include{ch_program_examples}
 
 \include{ch_jobscript_examples}
 
 \include{ch_best_practices}
 
-\include{ch_VNC}
+\ifthenelse{\NOT \boolean{antwerpen}}{%
```
Comment (boegel) [inline intro-HPC/intro-HPC.tex]: @sbecuwe Can you add a comment here to explain why this chapter is excluded for Antwerp?

File: intro-HPC/title.tex [hunk 2]
```diff
@@ -1,21 +1,15 @@
- \begin{center}
+\begin{center}
 
-\includegraphics*[width=8truecm]{logo_vsc}
+\includegraphics*[width=12truecm]{logo_vsc_2}
```
Comment (boegel) [inline intro-HPC/title.tex]: maybe rename `logo_vsc_2` to `logo_vsc_2018`?

File: intro-HPC/title.tex [hunk 3]
```diff
@@ -1,21 +1,15 @@
...
 
-\vspace*{1.5\baselineskip}
+\vspace*{6\baselineskip}
 
 \Huge \strong{HPC Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: September 25 2018
```
Comment (boegel) [inline intro-HPC/title.tex]: @sbecuwe Please keep `DATEPLACEHOLDER` here, this is automatically replaced by today's date when the PDF are generated by Travis.

File: intro-HPC/title.tex [hunk 4]
```diff
@@ -97,14 +92,39 @@
...
+
+\begin{tabular}{|p{\dimexpr 0.44\textwidth-2\tabcolsep}|>{\centering\arraybackslash}p{\dimexpr 0.12\textwidth-2\tabcolsep}|p{\dimexpr 0.44\textwidth-2\tabcolsep}|} \hline
+\multicolumn{3}{|c|}{\strong{Software-specific Best Practices Part}} \\ \hline
+MATLAB & \strong{\ref{ch:matlab}} & \nameref{ch:matlab} \\ \hline
+OpenFOAM & \strong{\ref{ch:openfoam}} & \nameref{ch:openfoam} \\ \hline
+Mympirun & \strong{\ref{ch:mympirun}} & \nameref{ch:mympirun} \\ \hline
+Singularity & \strong{\ref{ch:singularity}} & \nameref{ch:singularity} \\ \hline
+SCOOP & \strong{\ref{ch:scoop}} & \nameref{ch:scoop} \\ \hline
+Easybuild & \strong{\ref{ch:easybuild}} & \nameref{ch:easybuild} \\ \hline
+HOD & \strong{\ref{ch:hod}} & \nameref{ch:hod} \\ \hline
```
Comment (boegel) [inline intro-HPC/title.tex]: How are we going to remember to keep this up to date if new chapters are added?

File: intro-Linux/title.tex [hunk 5]
```diff
@@ -1,44 +1,36 @@
...
 
-\includegraphics*[width=8truecm]{logo_vsc}
+\includegraphics*[width=12truecm]{logo_vsc_2}
 
-\vspace*{1.5\baselineskip}
+\vspace*{6\baselineskip}
 
 \Huge \strong{Linux Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: September 20 2018
```
Comment (boegel) [inline intro-Linux/title.tex]: Same here, please keep `DATEPLACEHOLDER` (and maybe include a comment to make it clear it should stay)?

File: perfexpert/title.tex [hunk 6]
```diff
@@ -1,47 +1,38 @@
-\pagestyle{empty}
-
 \begin{center}
 
-\includegraphics*[width=8truecm]{logo_vsc}
+\includegraphics*[width=12truecm]{logo_vsc_2}
```
Comment (boegel) [inline perfexpert/title.tex]: `logo_vsc_2018`?

File: perfexpert/title.tex [hunk 7]
```diff
@@ -1,47 +1,38 @@
...
 
-\vspace*{1.5\baselineskip}
+\vspace*{6\baselineskip}
 
 \Huge \strong{PerfExpert Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
-
-\vspace*{1.5\baselineskip}
+\LARGE Last updated: September 14 2018
```
Comment (boegel) [inline perfexpert/title.tex]: Again, please keep `DATEPLACEHOLDER`.

File: intro-HPC/title.tex (hunk 2 above)
Comment (sbecuwe) [reply intro-HPC/title.tex]: Done.

File: intro-HPC/title.tex (hunk 3 above)
Comment (sbecuwe) [reply intro-HPC/title.tex]: Done. So we don't have to run add-timestamp.sh any longer?

File: perfexpert/title.tex (hunk 7 above)
Comment (sbecuwe) [reply perfexpert/title.tex]: Done.

File: perfexpert/title.tex (hunk 6 above)
Comment (sbecuwe) [reply perfexpert/title.tex]: Done.

File: intro-Linux/title.tex (hunk 5 above)
Comment (sbecuwe) [reply intro-Linux/title.tex]: Done.

File: intro-HPC/title.tex (hunk 4 above)
Comment (sbecuwe) [reply intro-HPC/title.tex]: I think we should have a consistent overview. grep or whatever could easily check whether the label of every chapter is present in title.tex ...

for i in `grep "\label{ch:" ch*tex | cut -f2- -d: | cut -c7-`; do if [[ `grep $i title.tex | wc -l` -ne 1 ]]; then echo "$i missing"; fi; done

File: intro-HPC/intro-HPC.tex (hunk 1 above)
Comment (boegel) [reply intro-HPC/intro-HPC.tex]: OK, but please mention that in a comment above this line, to avoid that others re-introduce the VNC chapter for UA.

Comment (sbecuwe) [reply intro-HPC/intro-HPC.tex]: Done. Also did it for checkpointing (gent and brussels).

File: intro-HPC/title.tex (hunk 4 above)
Comment (boegel) [reply intro-HPC/title.tex]: Done in https://github.com/hpcuantwerpen/vsc_user_docs/pull/2, please review & merge that PR to update this one.

Comment (sbecuwe) [reply intro-HPC/title.tex]: Looks good. Thanks!

**Pull request merged**